File: src/Jering.VectorArtKit.WebApi/ResponseModels/Account/LogInResponseModel.cs
using Jering.Mvc;
using Jering.VectorArtKit.WebApi.ResponseModels.Shared;
using Microsoft.AspNetCore.Mvc;

namespace Jering.VectorArtKit.WebApi.ResponseModels.Account
{
    public class LogInResponseModel: IErrorResponseModel
    {
        public bool AuthenticationError { get; set; }
        public bool AntiForgeryError { get; set; }
        public bool TwoFactorRequired { get; set; }
        public SerializableError ModelState { get; set; }
        public bool ExpectedError { get; set; }

        public string ErrorMessage { get; set; }
    }
}

File: src/jering-vectorartkit-webclient/src/api/response-models/log-in.response-model.ts



    export interface LogInResponseModel {
	    authenticationError?: boolean;
	    antiForgeryError?: boolean;
	    twoFactorRequired?: boolean;
	    modelState?: { [key: string]: any; };
	    expectedError?: boolean;
	    errorMessage?: string;
    }

    

        

